symfony/src/Service/MeasurementService.php

```diff
@@ -96,7 +96,16 @@ public function getMeasurmeent($participantId, $measurementId)
 
     public function requireBloodDonorCheck()
     {
-        return $this->params->has('feature.blooddonorpm') && $this->params->get('feature.blooddonorpm') && $this->session->get('siteType') === 'dv' && $this->siteService->isDiversionPouchSite();
+        return $this->params->has('feature.blooddonorpm') && $this->params->get('feature.blooddonorpm') && $this->session->get('siteType') === 'dv' && ($this->siteService->isDiversionPouchSite() || $this->isBloodDonorPmSite());
```

Comment:
I think we discussed this previously, but just wanted to double check - did we decide that if a site is not in a DV awardee that it shouldn't have the abbreviated PM form, even if the site id is included in the `blooddonorpmsites` list?

Reply:
Yes, I remember we decided to enable this only when the site's type is DV because the requirement is to have blood donor pm form and dv kit order workflow (which only applies when the site's type is DV)